sros2/sros2/api/_permission.py

```diff
@@ -0,0 +1,73 @@
+# Copyright 2016-2019 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+from lxml import etree
+
+import rclpy
```

Comment:
As we don't use anything else from rclpy, could we keep `from rclpy.utilities import get_rmw_implementation_identifier` instead?

Reply:
Of course, sorry, done.